src/exojax/spec/moldb.py

```diff
@@ -109,8 +110,7 @@ def __init__(self, path, nurange=[-np.inf, np.inf], margin=0.0, crit=0., Ttyp=10
         pf = exomolapi.read_pf(self.pf_file)
         self.gQT = jnp.array(pf['QT'].to_numpy())  # grid QT
         self.T_gQT = jnp.array(pf['T'].to_numpy())  # T forgrid QT
-        self.Tref = 296.0
```

Comment:
self.Tref -> Tref in [line 765 (in func. qr_iso_interp)](https://github.com/HajimeKawahara/exojax/blob/c476903f8c2034d0b793a461b7dcc033be173a74/src/exojax/spec/moldb.py#L765)